packages/core/src/components/Steps/StepsGalleryHeader.tsx

```diff
@@ -9,7 +10,7 @@ export interface StepsGalleryHeaderProps extends VibeComponentProps {
   activeStepIndex: number;
   stepsCount: number;
   onChangeActiveStep: (e: React.MouseEvent, stepIndex: number) => void;
-  stepDescriptionFunc: (stepIndex: number) => string;
+  stepDescriptionFunc?: (stepIndex: number) => string;
```

Comment:
I'm not really fond of this name

Reply:
lol
this should be re-written, I don't like the way this component works